samples/core/src/test/resources/BillExportJobTest/expected/bills_pg.csv

```diff
@@ -0,0 +1,6 @@
+1,true,14.98 EUR,1.30 EUR,1,1,,1,Schnitzel-Menü,DELIVERED,6.99 EUR,mit Ketschup
```

Comment:
@anmaso , JUNITs (mvn clean install) were failing because of the duplicate rows in the current csv file . Hence , created a different bills.csv for PostGres removing the duplicate rows. 

Also, just to keep you informed , we are using the branch 'develop-2.3.0' for the next milestone 'oasp4j 2.3.0'. Hence while merging please merge the files onto the branch 'develop-2.3.0'

Thanks,
Kiran.